infrastructure/versions.tf

```diff
@@ -0,0 +1,23 @@
+# Configure the Azure provider
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.5.0"
+    }
+  }
+  required_version = ">= 1.1.0"
+
+  backend "azurerm" {}
+  #backend "local" {
```

Comment:
If we want to keep this commented out - perhaps a quick one-liner to indicate its purpose?